packages/dcos-net/extra/gen_resolvconf.py

```diff
@@ -37,7 +38,7 @@ def check_server(addr):
     except dns.exception.Timeout:
         print('Skipping DNS server {}: no response'.format(
             addr), file=sys.stderr)
-    except:
+    except Exception:
         print("Unexpected error querying DNS for server \"{}\" exception: {}".format(
             addr, sys.exc_info()[1]))
```

Comment:
I know it's not a part of diff but is this a proper way of printing exceptions?

Reply:
Yea not really, doing:

```python
except Exception as e:
    print("Unexpected error querying DNS for server \"{}\" exception: {}".format(addr, e))
```

Would be the right way™. After looking into this script, ugh, it should be heavily rewritten. I was just fixing the flake8 linting error of bare except ;).